app/models/assignment.rb

```diff
@@ -18,4 +18,10 @@ class Assignment < ActiveRecord::Base
   delegate :department, to: :course, prefix: true
 
   validates :name, presence: true
+
+  has_paper_trail
+
+  def subject_name
+    "#{subject.number} - #{subject.title}"
```

Comment:
This is the definition of `subject.to_s`. What do you think of calling that here instead?